.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Should we not also include the actual generated hooks? https://pre-commit.com/#3-install-the-git-hook-scripts

Reply:
If by "include" you mean installing the hooks on your own machine, that would be the idea.

Reply:
No, I mean including the hooks on this repo by default so they'll run without any configuration. I have no experience with pre-commit hooks though, so not sure if that's possible?

Reply:
To my knowledge this isn't possible. Hooks are client side but need to be enabled/installed by the user.
They can't be committed either. You could inject commands into anyone that clones the repository if this would be possible.

There are hooks that can be setup server but they require privileged access.